docs/conceptual/gpu-arch/mi200-performance-counters.md

```diff
@@ -1,455 +1,591 @@
+| Hardware counter   | Unit   | Definition |
+| :--------------------| :--------| :-------------------|
+| `GRBM_COUNT` | Cycles | Number of free-running GPU cycles |
+| `GRBM_GUI_ACTIVE` | Cycles | Number of GPU active cycles |
+| `GRBM_CP_BUSY` | Cycles | Number of cycles any of the command processor (CP) blocks are busy |
+| `GRBM_SPI_BUSY` | Cycles | Number of cycles any of the shader processor input (SPI) are busy in the shader engine(s) |
+| `GRBM_TA_BUSY` | Cycles | Number of cycles any of the texture addressing Unit (TA) are busy in the shader engine(s) |
+| `GRBM_TC_BUSY` | Cycles | Number of cycles any of the texture cache (TC) blocks (TCP/TCI/TCA/TCC) are busy |
+| `GRBM_CPC_BUSY` | Cycles | Number of cycles the command processor - compute (CPC) is busy |
+| `GRBM_CPF_BUSY` | Cycles | Number of cycles the command processor - fetcher (CPF) is busy |
+| `GRBM_UTCL2_BUSY` | Cycles | Number of cycles the unified translation cache - level 2 (UTCL2) block is busy |
+| `GRBM_EA_BUSY` | Cycles | Number of cycles the efficiency arbiter (EA) block is busy |
 
 ### Command processor
 
-The command processor counters are further classified into fetcher and compute.
+The command processors (CPs) are further classified into fetcher (CPF) and compute (CPC).
 
 #### CPF
 
 ##### CPF counters
 
-| Hardware Counter                     | Unit   | Definition                                                   |
-|--------------------------------------|--------|--------------------------------------------------------------|
-| `cpf_cmp_utcl1_stall_on_translation` | Cycles | One of the compute UTCL1s is stalled waiting on translation. |
-| `cpf_cpf_stat_idle[∗]`               | Cycles | CPF idle                                                   |
-| `cpf_cpf_stat_stall`                 | Cycles | CPF stall                                                  |
-| `cpf_cpf_tciu_busy`                  | Cycles | CPF TCIU interface busy                                    |
-| `cpf_cpf_tciu_idle`                  | Cycles | CPF TCIU interface idle                                    |
-| `cpf_cpf_tciu_stall[∗]`              | Cycles | CPF TCIU interface is stalled waiting on free tags.        |
+| Hardware counter | Unit   | Definition |
+| :-----------------------| :--------| :------------------------|
+| `CPF_CMP_UTCL1_STALL_ON_TRANSLATION` | Cycles | Number of cycles one of the compute UTCL1s is stalled waiting on translation |
+| `CPF_CPF_STAT_BUSY` | Cycles | Number of cycles CPF is busy |
+| `CPF_CPF_STAT_IDLE[∗]` | Cycles | Number of cycles CPF is idle |
+| `CPF_CPF_STAT_STALL` | Cycles | Number of cycles CPF is stalled |
+| `CPF_CPF_TCIU_BUSY` | Cycles | Number of cycles CPF TCIU interface is busy |
+| `CPF_CPF_TCIU_IDLE` | Cycles | Number of cycles CPF TCIU interface is idle |
+| `CPF_CPF_TCIU_STALL[∗]` | Cycles | Number of cycles CPF TCIU interface is stalled waiting on free tags |
 
 #### CPC
```

Comment:
@SwRaw this is an empty heading; We should add content here or remove the following heading (CPC counters).